components/frontend-react/src/containers/Dashboard.js

```diff
@@ -61,27 +63,25 @@ const styles = theme => ({
 
 class Dashboard extends React.Component {
 
-  constructor(){
-    super();
+  constructor(props){
+    super(props);
```

Comment:
A fancy way of doing this is
```
class Dashboard extends React.Component {
  state = {
    link: '',
    completedList:[]
  }
```
and then remove the constructor.

I think I remember it correctly, please correct me if I'm wrong. @iamchathu .

Reply:
@agentmilindu With `classProperties` syntax support now we don't want to use constructor to initialize state.